src/main/java/com/google/firebase/messaging/Message.java

```diff
@@ -68,19 +68,25 @@ private Message(Builder builder) {
     this.androidConfig = builder.androidConfig;
     this.webpushConfig = builder.webpushConfig;
     this.apnsConfig = builder.apnsConfig;
+
+    String unprefixedTopic = null;
```

Comment:
Can you move this to a helper function and call this stripPrefix()? I think the name unprefixedTopic will never win my heart over :)

Reply:
Done :+1: